README.md

```diff
@@ -115,7 +126,33 @@ git clone https://github.com/uclapi/uclapi
 git clone https://github.com/uclapi/fakeshibboleth
 ```
 
+### Quick Note on using virtual environments
+
+Virtual environments allow us to easily switch between python versions and the pip packages we have installed. They are entered with the following commands:
+
+```
+Linux: . venv/bin/activate 
+Windows: \venv\Scripts\activate
+```
+
+You can tell you're in one by the first letters of a terminal/cmd prompt. The line should start with the name of your virtual environment for example 
+
+```
+(venv)Your-Computer:your_project UserName$ 
+```
+
+Once inside anything we run runs with this environments version of python. This must be done whenever using our project and so before running *manage.py* or other python files remember to be in the virtual environment.
```

Comment:
*environment's version of Python, and the versions of any packages we have installed within it.